### PR TITLE
[Workers] Update workers.yaml

### PR DESCRIPTION
**Pull request by ohodson**

### CODE REVIEW

**Comment by Nana250999** [comment]
> _ไม่มีคำอธิบาย_

is:pr
is:issue

**Review by Nana250999 — COMMENTED** [review]
is:pr
is:issue

**Comment by Nana250999** [comment]
> _ไม่มีคำอธิบาย_

is:pr
is:issue

**Comment by ohodson** [comment]
Replacement for https://github.com/cloudflare/cloudflare-docs/pull/11284 that seemed to get immovably stuck in checks.

File: data/changelogs/workers.yaml [hunk 1]
```diff
@@ -16,6 +16,13 @@ entries:
   description: |-
     - The limit of 3 Cron Triggers per Worker has been removed. Account-level limits on the total number of
       Cron Triggers across all Workers still apply.
+- publish_date: '2023-10-12'
+  description: |-
+    - A [TCP Socket](/workers/runtime-apis/tcp-sockets/)'s WritableStream now ensures the connection has opened before resolving the promise returned by `close`.
+- publish_date: '2023-10-09'
+  description: |-
+    - The Web Platform standard [`CustomEvent` class](https://dom.spec.whatwg.org/#interface-customevent) is now available in Workers.
+    - Fixed a bug in the WebCrypto API where the `publicExponent` field of the algorithm of RSA keys would have the wrong type. Use the `cryptoPreservePublicExponent` compatibility flag to enable the new behavior.
```
Comment (KianNH) [inline data/changelogs/workers.yaml]:
```suggestion
    - Fixed a bug in the WebCrypto API where the `publicExponent` field of the algorithm of RSA keys would have the wrong type. Use the [`crypto_preserve_public_exponent` compatibility flag](/workers/configuration/compatibility-dates/#webcrypto-preserve-publicexponent-field) to enable the new behavior.
```

**Review by Nana250999 — COMMENTED** [review]
[__](`

> #2693 

`)

**Review by deadlypants1973 — APPROVED** [review]

**Comment by Nana250999** [comment]
> _ไม่มีคำอธิบาย
branch_protection.yaml
binary_artifacts.yaml
outside.yaml

**Pull request merged**